src/libs/actions/IOU.js

```diff
@@ -563,6 +583,15 @@ function createSplitsAndOnyxData(participants, currentUserLogin, currentUserAcco
             '',
             oneOnOneIOUReport.reportID,
         );
+        // Add optimistic personal details for new participants
+        const oneOnOnePersonalDetailListAction = {
+            [accountID]: {
+                accountID,
```

Comment:
Did we optimistically create the accountID for the split bill participants who do not exist?

Reply:
Yes, they should already have been generated in `OptionsListUtils` when the user searches for the new user.

Reply:
Thanks!